main.cpp

```diff
@@ -1,162 +1,190 @@
+	for each of those functions declared,
+		a) write out how the function would look if called with correct arguments 
+        b) if the function returned anything, store it in a local va able via the 'auto' keyword. 
+        c) pass the local variables to ignoreUnused() as  you did in 2b) see main() for an example of this.
+
+10) click the [run] button.  Clear up any errors or warnings as best you ca
+.
+*/
 
-//2)
-void variableDeclarations()
+// 2)
+void variableDeclarations() 
 {
-    //example:
-    int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
+	// example:
+    int number = 2; // declaration of a variable named "number", that uses the type 'int', and the variable's initial value is '2'
+    int myAge = 2;
+    int numberOfParticipants = 0;
+    int numberOfPlayers = 10;
+    double shoeSize = 6.5;
+    double fuelPrice = 2.1;
+    double heightRoom = 3.1;
+    bool lightIsOn = false;
+    bool coffeeIsPrepared = true;
+    bool waterIsBoiled = true;
+    float timeToLaunchRocket = 42.2f;
+    float hairThickness = 1.5f;
+    float statueHeight = 10005.32f;
+    unsigned int numberOfHousesInTown = 452;
+    unsigned int daysLeftToHolidays = 32;
+    unsigned int numberOfPetsAtHome = 3;
     
+    ignoreUnused(number, myAge,numberOfParticipants, numberOfPlayers, shoeSize, fuelPrice, heightRoom, lightIsOn, coffeeIsPrepared, waterIsBoiled, timeToLaunchRocket, hairThickness, statueHeight, numberOfHousesInTown, daysLeftToHolidays, numberOfPetsAtHome);
+        
+	// passing each variable dec to the ignoreUnused()
 
-    
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+	// function
 }
 
 /*
  10 functions
  example:
- note: this example shows the result after completing steps 3-8
+ note: this example shows the result after completing sps 3-8
  */
-bool rentACar(int rentalDuration, int carType = 0)  //function declaration with random number of arguments, arbitrary number of arguments have default value
-{ 
-    ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused() function
-    return {}; //if your function returns something other than void, add 'return {};' at the end of it.
-} 
+bool rentACar(int rentalDuration, int carType = 0) // function declaration wi number of arguments,  arbitrary number of arguments have default value
+{
+    ignoreUnused(rentalDuration, carType); // pass  parameter to the ignoreUnused() function
+    return {}; // if your function returns  other than void, add
+               // 'return {};' at the end of it.
+}
 
-/*
- 1)
- */
+bool buyAnotherPlant(int spaceForPlantsAtHome, int numberOfPlantsAtHome) 
+{
+    ignoreUnused(spaceForPlantsAtHome, numberOfPlantsAtHome);
+    return {};
+}
 
-/*
- 2)
- */
+int daysLeftToSaturday(int numberOfTheDayInTheWeek) 
+{
+    ignoreUnused(numberOfTheDayInTheWeek);
+    return {};
+}

```

Comment:
This function name does not conform to the programming guideline: 
```
Name Things what they are.  
Name Actions based on what they do
```

The name you chose reflects a property (noun) and does not contain a verb, which would reflect an action.
Functions perform work, so their names must contain a verb which describes the work being performed.
Examples of function names that contain verbs:
- `computeAValue()`
- `estimateDistanceTraveled()`
- `performDiagnostics()` 

- [ ] revise